README.md

```diff
@@ -54,10 +54,10 @@ What happens if the update fails?
   ls $ANDROID_SDK_ROOT/tools/
   #=> empty, nothing is there
   # tools such as: android, sdkmanager, emulator, lint and etc. are gone
-  
+
```

Comment:
Please remove all reformatting changes, the indentation is required by GitHub's editor.